### DISCUSSION TITLE
dbatools command equivalent of Oracle DESCRIBE

### DISCUSSION THREAD

**Discussion by Liturgist** [dataplat/dbatools#7339] [Q&A] [answered]
Is there a dbtools command that will produce the equivalent of the Oracle DESCRIBE command? Or, a command that would produce the fields from which it could be produced.

Is there a command that would produce DDL in a manner similar to SSMS Script Table as > CREATE to?

Get-DbaDbTable does not do it.

**Comment by niphlod** [comment, marked as answer]
see https://docs.dbatools.io/#Export-DbaScript
